ios/RNRadar.m

```diff
@@ -238,13 +219,32 @@ - (void)didLogMessage:(NSString *)message {
         rejecter = nil;
     };
 
+    NSString *accuracy = optionsDict[@"accuracy"];
+
+    if (accuracy != nil && [accuracy isKindOfClass:[NSString class]]) {
+        if ([accuracy isEqualToString:@"high"] || [accuracy isEqualToString:@"HIGH"]) {
+            desiredAccuracy = RadarTrackingOptionsDesiredAccuracyHigh;
+        } else if ([accuracy isEqualToString:@"medium"] || [accuracy isEqualToString:@"MEDIUM"]) {
+            desiredAccuracy = RadarTrackingOptionsDesiredAccuracyMedium;
+        } else if ([accuracy isEqualToString:@"low"] || [accuracy isEqualToString:@"LOW"]) {
+            desiredAccuracy = RadarTrackingOptionsDesiredAccuracyLow;
+        }
+    }
+
+    BOOL beacons = optionsDict[@"beacons"];
+
+    if (beacons) {
+        beaconsTrackingOption = beacons;
+    }
+
```

Comment:
If `beaconsTrackingOption` isn't used elsewhere, these lines can be moved into the `else` clause below.